frontend/src/components/Main.js

```diff
@@ -1,17 +1,19 @@
-import { Fragment }from 'react'
+import { Fragment, useState }from 'react'
 import SearchForm from './SearchForm';
 import ResultsList from './ResultsList';
 import '../sass/components/main.scss';
 
 function Main() {
-  return <Fragment>
+    const [radioDifficulty, setRadioDifficulty] = useState('beginner');
+    return (
+        <Fragment>
             <main className='main'>
             <h1 className='heading-primary'>abb - always be building</h1>
-
-                <SearchForm />
-                <ResultsList />
+                <SearchForm radioDifficulty={radioDifficulty} setRadioDifficulty={setRadioDifficulty}/>
+                <ResultsList radioDifficulty={radioDifficulty}/>
```

Comment:
Since you're no longer sorting, passing state as props to this component is no longer necessary

Reply:
That's true! Will clear this up.